integration-tests/bats/sql-server.bats

```diff
@@ -1258,7 +1258,7 @@ END""")
     # can't drop a branch-qualified database name
     run server_query "" 1 "drop database \`test2/newbranch\`"
     [ "$status" -ne 0 ]
-    [[ "$output" =~ "database not found: test2/newbranch" ]] || false
+    [[ "$output" =~ "unable to drop revision database: test2/newbranch" ]] || false
 
     server_query "" 1 "drop database TEST2"
```

Comment:
Would be cool to have some bats tests showing the output for `show databases` after something like `use mydb/HEAD~`